terratest/run-tests-from-cmdline.md

```diff
@@ -1,19 +1,11 @@
 # Running tests from the command line
 
-The test can also be run from the command line which allows you to avoid complying with the IntelliJ plugin license requirements.
+The tests can be run from the command line which allows you to avoid complying with the IntelliJ plugin license requirements.
```

Comment:
rather than sounding naughty, how about this?
Run the tests using the command line to avoid using the IntelliJ plugin.